src/app/missions/detail/detail.component.html

```diff
@@ -65,11 +65,15 @@ <h3 id="general">General</h3>
   </table>
 </section>
 
-<app-icon class="mission-icon" [iconID]="mission.missionIconID" *ngIf="mission && mission.missionIconID"></app-icon>
+<section id="mission-display">
```

Comment:
Yeah, as the main comment says, this takes up (much?) more space than before. Consider moving it.